pinax/messages/tests/tests.py

```diff
@@ -233,6 +233,72 @@ def test_unread(self):
             "READ",
         )
 
+    def test_unread_threads_one_unread(self):
+        """
+        Ensure `unread_threads` template_tag produces correct results for one unread message
+        """
+        thread = Message.new_message(
+            self.brosner,
+            [self.jtauber],
+            "Why did you break the internet?", "I demand to know.").thread
+
+        tmpl = """
+               {% load pinax_messages_tags %}
+               {% if user|unread_threads %}{{ user|unread_threads }}{% endif %}
+               """
+        self.assert_renders(
+            tmpl,
+            Context({"thread": thread, "user": self.jtauber}),
+            '1'
+        )
+
+    def test_unread_threads_two_messages_incl_reply(self):
+        """
+        Ensure `unread_threads` template_tag produces correct results.
+        """
+        thread = Message.new_message(
+            self.brosner,
+            [self.jtauber],
+            "Why did you break the internet?", "I demand to know.").thread
+
+        Message.new_reply(thread, self.jtauber, "Replying to the first message")
+        Message.new_reply(thread, self.brosner, "Replying again, so that there are two unread messages on one thread")
+
+        thread = Message.new_message(
+            self.brosner,
+            [self.jtauber],
+            "Second message", "So there are two.").thread
+        tmpl = """
+               {% load pinax_messages_tags %}
+               {% if user|unread_threads %}{{ user|unread_threads }}{% endif %}
+               """
+        self.assert_renders(
+            tmpl,
+            Context({"thread": thread, "user": self.jtauber}),
+            '2'
+        )
```

Comment:
I think it would be smart to also check the number of Threads with unread messages for "brosner" in this test. I believe the value should be "1".